docs/docs/ref-01-top-level-api.md

```diff
@@ -132,6 +132,10 @@ array React.Children.toArray(object children)
 
 Return the `children` opaque data structure as a flat array with keys assigned to each child. Useful if you want to manipulate collections of children in your render methods, especially if you want to reorder or slice `this.props.children` before passing it down.
 
+> Note:
+> 
+> `React.Children.toArray()` changes the key so that the identity semantics can be preserved. That is, the keys are altered to ensure they do not collide with any siblings that may have the same key.
```

Comment:
Can we make a small tweak and do:

> `React.Children.toArray()` changes keys to preserve the semantics of nested arrays when flattening lists of children. That is, `toArray` prefixes each key in the returned array so that each element's key is scoped to the input array containing it.

Thanks!